src/components/Blogs/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Blogs renders correctly and matches the snapshot 1`] = `
+<div
+  className="container"
```

Comment:
this test should show at least one mock blog post. you will have to mock fetch so that it returns some blog post data. google "mock fetch jest unit test" or work with ammar in this one. but it will be worth your time to do this, testing is by far the most popular interview question I've seen from companies so consider yourself lucky to get the chance to do this one😀

Reply:
I always thought I am the lucky one, now I am sure that I am :)